package.json

```diff
@@ -88,6 +88,7 @@
     "@sentry/browser": "^5.0.8",
     "@toruslabs/torus-embed": "^1.3.0",
     "@walletconnect/web3-provider": "^1.0.0-beta.37",
+    "apollo-link-retry": "^2.2.16",
```

Comment:
Why add this?

Reply:
Actually it's not necessary. For some reason when targeting arc.js locally to Alchemy (in order to test) I was asked to install that dependency, otherwise it won't compile. I forgot to remove this, will do.